src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -666,6 +666,11 @@ public CitizenData getCitizenData()
     @Override
     public boolean hasCapability(final Capability<?> capability, final EnumFacing facing)
     {
+        if (getCitizenData() == null)
+        {
+            return false;
+        }
+
```

Comment:
yeah, that isnt working at all, i cant interact with any of the core blocks for buildings, and all my citizens are MIA

Reply:
that's because your colony got corrupted, load a backup of minecolonies.dat
it happens sometimes when you don't quit properly or crash

Reply:
```
-rw-r--r-- 1 clever users 31K Jan 10 02:21 colonies-2018-01-10_02.21.24.dat
-rw-r--r-- 1 clever users 31K Jan 10 02:23 colonies-2018-01-10_02.23.31.dat
-rw-r--r-- 1 clever users 137 Jan 11 03:44 colonies.dat
```
yeah, that size isnt right

Reply:
yeah, after restoring that, its trying to downgrade half the buildings in my test world, and i can confirm the above patch solves the issue

Reply:
yep =D just use on of the backups and it'll load fine

Reply:
if you consider it turning a level 5 into a level 3 "fine", could use more frequent backups, though its just a test world where i cheated it all in